### ISSUE TITLE
Alternate default browser issues

### ISSUE THREAD

**Issue by snevas** [privacybydesign/irmamobile#15]
When trying to load attributes e.g. with DigiD with a alternate default browser set, you could end up with a redirect to the app store webpage. This is because e.g. Firefox does not open app-links by default (privacy measure), which triggers the redirect. This could be solved within the app (WebView), or within the Gemeente Nijmegen-flow.

Version app: 6.0.11
Alternate default browser: Firefox Mobile

Steps:
- Open IRMA app
- Voeg meer gegevens toe
- Klik woonadres / persoonsgegevens
- Voeg toe
- Inloggen met DigiD
- You end up with the webview of the irma app, app store webpage.

**Comment by ivard** [comment]
Yeah we know about this inconvenience. We cannot solve it in the webview either, since we use custom tabs (which is just the normal browser version with some theming). Webviews are not properly implemented yet in Flutter. I'd still want to convert the links on websites from web links (universal links) to direct app links without a browser fallback. The only thing we have to solve for this is adding some handling for users that don't have the IRMA app yet. Of course they cannot click on direct app links if they haven't installed the app.

So it is on our list, but it is a bit more work than you might expect :)

**Comment by snevas** [comment]
Okay, but what if you would not do a automatic redirect to the appstore in the Nijmegen-flow, but make a landing page with 2 buttons:
- Open in IRMA app (which Firefox allows in button form, as it's a user decision)
- Install IRMA app (link to appstore)

This is also how other parties like ABN-amro handle this with payment links.

You end up with a sitation where for 90% of people it would still work as before and the other 10% would end up in this landing page where you could still load the attibute.

**Comment by ivard** [comment]
We actually do so. Some issuers however still use an older version of our library, so that's why it is not visible there. For example if you try the [AngryGames demo](https://angrygames.nl/) you'll see that we don't do automatic redirects anymore.

The only thing indeed that we still have to add to the `irma-frontend-packages` (it's on my wish list for some time) that users get the option to install the IRMA app if they haven't done that already. That view in the UX is still missing there. The IRMA app perfectly supports this scenario, so here we don't need extra functionality. I'll therefore move this issue to there.
